docs/src/docs/hf-import.md

```diff
@@ -0,0 +1,21 @@
+# Importing a ModelKit from Hugging Face
+
+You can automatically generate a ModelKit from a Hugging Face repository using the `kit import` command. This speeds and simplifies the job of getting started with ModelKits.
+
+::: info
+In v1.0.0 Git LFS is required for importing from Hugging Face
+:::
+
+You can read more about the `import` command in our [CLI reference](https://kitops.ml/docs/cli/cli-reference/#kit-import).
+
+## Importing a Hugging Face repository
+
+1. **Get the HF URL**: On the Hugging Face site, copy the URL from the repository you want to create a ModelKit from (e.g., https://huggingface.co/HuggingFaceTB/SmolLM-135M-Instruct). You can also customize the name or add a tag name if desired.
+
+1. **Kit import**: In a terminal window running Kit version 1.0.0 at least, type `kit import https://huggingface.co/HuggingFaceTB/SmolLM-135M-Instruct`. This will download and build a [Kitfile](./kitfile/kf-overview.md) based on the Hugging Face model and give you an opportunity to edit it before the ModelKit is packed.
```

Comment:
I guess this can be confusing, but part of the implementation was to simplify things: both

```
kit import https://huggingface.co/HuggingFaceTB/SmolLM-135M-Instruct
```
and
```
kit import HuggingFaceTB/SmolLM-135M-Instruct
```
are equivalent (i.e. you don't need the huggingface bit)